PeriodicSchedulerImplTest unstable

This test failed in: https://ci.openhab.org/job/PR-openHAB-Core/4378/

```
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.829 s <<< FAILURE! - in org.openhab.core.internal.scheduler.PeriodicSchedulerImplTest
[ERROR] org.openhab.core.internal.scheduler.PeriodicSchedulerImplTest.testSchedule  Time elapsed: 1.708 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected periodic time ==> expected: <4> but was: <3>
        at org.openhab.core.internal.scheduler.PeriodicSchedulerImplTest.testSchedule(PeriodicSchedulerImplTest.java:60)
```

--- a/bundles/org.openhab.core/src/test/java/org/openhab/core/internal/scheduler/PeriodicSchedulerImplTest.java
+++ b/bundles/org.openhab.core/src/test/java/org/openhab/core/internal/scheduler/PeriodicSchedulerImplTest.java
@@ -47,7 +47,7 @@ public class PeriodicSchedulerImplTest {
         final long now = System.currentTimeMillis();
 
         ScheduledCompletableFuture<Object> future = periodicScheduler.schedule(() -> {
-            times.add((System.currentTimeMillis() - now) / 100);
+            times.add(System.currentTimeMillis() - now);
             semaphore.release();
         }, delays);
         semaphore.acquire(6);
@@ -55,9 +55,10 @@ public class PeriodicSchedulerImplTest {
         // Because starting scheduler takes some time and we don't know how long
         // the first time set is the offset on which we check the next values.
         long offset = times.poll();
-        long[] expectedResults = { 2, 5, 8, 11, 14 };
+        long[] expectedResults = { 200, 500, 800, 1100, 1400 };
         for (long expectedResult : expectedResults) {
-            assertEquals(offset + expectedResult, times.poll().longValue(), "Expected periodic time");
+            assertEquals((offset + expectedResult) / 100.0, times.poll().longValue() / 100.0, 0.1,
+                    "Expected periodic time");
         }
         assertFalse(semaphore.tryAcquire(1, TimeUnit.SECONDS), "No more jobs should have been scheduled");
     }

--- a/bundles/org.openhab.core/src/test/java/org/openhab/core/internal/scheduler/PeriodicSchedulerImplTest.java
+++ b/bundles/org.openhab.core/src/test/java/org/openhab/core/internal/scheduler/PeriodicSchedulerImplTest.java
@@ -55,10 +55,12 @@ public class PeriodicSchedulerImplTest {
         // Because starting scheduler takes some time and we don't know how long
         // the first time set is the offset on which we check the next values.
         long offset = times.poll();
-        long[] expectedResults = { 200, 500, 800, 1100, 1400 };
+        long[] expectedResults = { 200, 300, 300, 300, 300 };
         for (long expectedResult : expectedResults) {
-            assertEquals((offset + expectedResult) / 100.0, times.poll().longValue() / 100.0, 0.3,
-                    "Expected periodic time");
+            long actualValue = times.poll().longValue();
+            assertEquals((offset + expectedResult) / 100.0, actualValue / 100.0, 0.3,
+                    "Expected periodic time, total: " + actualValue);
+            offset = actualValue;
         }
         assertFalse(semaphore.tryAcquire(1, TimeUnit.SECONDS), "No more jobs should have been scheduled");
     }

--- a/bundles/org.openhab.core/src/test/java/org/openhab/core/internal/scheduler/PeriodicSchedulerImplTest.java
+++ b/bundles/org.openhab.core/src/test/java/org/openhab/core/internal/scheduler/PeriodicSchedulerImplTest.java
@@ -57,7 +57,7 @@ public class PeriodicSchedulerImplTest {
         long offset = times.poll();
         long[] expectedResults = { 200, 500, 800, 1100, 1400 };
         for (long expectedResult : expectedResults) {
-            assertEquals((offset + expectedResult) / 100.0, times.poll().longValue() / 100.0, 0.1,
+            assertEquals((offset + expectedResult) / 100.0, times.poll().longValue() / 100.0, 0.3,
                     "Expected periodic time");
         }
         assertFalse(semaphore.tryAcquire(1, TimeUnit.SECONDS), "No more jobs should have been scheduled");

